Core/src/main/java/com/paypal/android/core/CoreConfig.kt

```diff
@@ -1,5 +1,13 @@
 package com.paypal.android.core
 
+/**
+ * Configuration for SDK requests.
+ *
+ * @property clientId your client ID from the PayPal Developer Portal
+ * @property clientSecret
```

Comment:
It seems like this should have a description as well, akin to `clientId` above

Reply:
there is a TODO in this class, that consist on deleting the `clientSecret`. We added it at the beginning as a temporary fix por authenticating api calls until `ordersV2` didnt need it anymore. I think they have fix it, so we might have to write a ticket for this.